config.json

```diff
@@ -1,6 +1,6 @@
 {
   "command_channels": [
-    1192895746672951340
+    1194820338933043271
```

Comment:
Don't change the production config for personal testing! Instead, create your own config and use the `--config` argument when launching the bot.